src/java.compiler/share/classes/javax/lang/model/util/Types.java

```diff
@@ -270,6 +270,8 @@ WildcardType getWildcardType(TypeMirror extendsBound,
      * for example, this method may be used to get the
```

Comment:
While unrelated to this change, if you cam, I would suggest moving `for example` to the beginning of the sentence, to give the reader more context for the text that follows:

> For example, given the type element for {@code Set} and a type mirror for {@code String},
> this method may be used to get the parameterized type {@code Set<String>}.